From 62884b84c5b369fff1f7be30dc961e6dfaa95b22 Mon Sep 17 00:00:00 2001
From: Unknown <unknown>
Date: Tue, 8 May 2007 20:28:06 +0000
Subject: [PATCH 1/3] Remove dbus API abuse

The use of dbus_connection_close prevents a normal user from using
pumount if a device was mounted with pmount-hal.

Origin: Gentoo
Upstream-Status: Unknown
[ismael@sourcemage.org: Edited description]
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 src/pmount-hal.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/pmount-hal.c b/src/pmount-hal.c
index 64acf48113e5..b185c1ae0089 100644
--- a/src/pmount-hal.c
+++ b/src/pmount-hal.c
@@ -401,7 +401,6 @@ main( int argc, const char** argv )
     /* shut down hal connection */
     libhal_ctx_shutdown( hal_ctx, &error );
     libhal_ctx_free( hal_ctx );
-    dbus_connection_close( dbus_conn );
     dbus_connection_unref( dbus_conn );
 
     /* go */
-- 
2.40.1

